src/SFA.DAS.EmployerFeedback.Api/AppStart/AddServiceRegistration.cs

```diff
@@ -19,14 +20,16 @@ public static void AddServices(this IServiceCollection services)
             services.AddMediatR(cfg => cfg.RegisterServicesFromAssembly(typeof(GetAttributesQuery).Assembly));
 
             services.AddValidatorsFromAssemblyContaining<SubmitEmployerFeedbackCommand>();
+            services.AddValidatorsFromAssemblyContaining<UpsertSettingsCommand>();
```

Comment:
It is only necessary to include a single reference the assembly which contains the validation - all validators will be found